### ISSUE TITLE
Lights change but order stays the same for player inputs

### ISSUE THREAD

**Issue by Vodkanakas** [pyhammond/retropie_wiimote_lights#3]
So I have 3 controllers. Connect 1 controller 1 light shows up. Connect second controller 1 lights up and first controller now has 2 lights. Connect third controller 1 light shows up, second controller shows 2 lights and first controller shows 3 lights. But if I check js0 inputs it's still the first controller connected (js1 p2, js2 p3) and if I go into a game js0 is p1 and so forth. Any way to get the lights corrected?

Pi4
4.5.12 nightly image

**Comment by pyhammond** [comment]
Yes, run the installer again and pick the "Internal" option rather than the "External" option.  The "External" option causes the order to be reversed because many external USB Bluetooth controllers flip them around for some reason.

If that doesn't work, then try the other way.  One will be in the reverse order of the other.

-Peter
